config/arch/xtensa.in

```diff
@@ -35,3 +35,7 @@ config ARCH_xtensa_fsf
     bool "fsf - Default configuration"
 
 endchoice
+
+if XTENSA_CUSTOM
+source "config/overlay.in"
```

Comment:
Since overlay now has a separate Kconfig fragment, can you move `TARGET_USE_OVERLAY` into it? You can then just define it as a `def_bool y`, so that the mere inclusion of overlay.in sets it, with no need for `select TARGET_USE_OVERLAY`. I don't like that there are two knobs for enabling the overlays, selecting this option and inclusion of the Kconfig fragment.

Reply:
Sure.